### PR TITLE
fix(DB/Dungeon/Quest): Entry into Karazhan

### PR DESCRIPTION
**Pull request by wizzymore**
## CHANGES PROPOSED:
-  Fix the spawn of the "First Fragment Guardian" for the quest Entry into Karazhan

## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3033

## TESTS PERFORMED:
- Spawned the creature, it spawned correctly now

## HOW TO TEST THE CHANGES:

.go gob 22674
.quest add 9831
Right click on the jar
The NPC should spawn on top of the jar, like here: https://www.youtube.com/watch?v=SHHmDpLSFZw
If you can't execute the query, keep in mind the Safe Mode of your client needs to be disabled ( as event_scripts table doesn't have any primary key )

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1601868446381471500.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1601868446381471500');
+
+DELETE FROM `event_scripts` WHERE `id`=14592;
+INSERT INTO `event_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `dataint`, `x`, `y`, `z`, `o`) VALUES (14592, 1, 10, 22890, 300000, 0, -108.252, -510.302, 21.4761, 2.44346);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1601868446381471500.sql:4]:
```suggestion
INSERT INTO `event_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `dataint`, `x`, `y`, `z`, `o`) VALUES 
(14592, 1, 10, 22890, 300000, 0, -108.252, -510.302, 21.4761, 2.44346);

```

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by Remberance** [comment]
will summon 2 mobs with quest 9831?

**Comment by wizzymore** [comment]
No, it spawns only one on the location of the chest.

Make sure the query is properly applied and the old event was deleted

**Comment by Remberance** [comment]
worked

**Pull request merged**